docker-for-windows/faqs.md

```diff
@@ -239,64 +212,58 @@ To learn more about how to set the client TLS certificate for verification, see
 [What to know before you install](install#what-to-know-before-you-install) in the
 Getting Started topic. For more about Docker Machine itself, see
 [What is Docker Machine?](/machine/overview#what-is-docker-machine), and the
 [Hyper-V driver](/machine/drivers/hyper-v) for Docker Machine.
 
 ## Windows Requirements
 
-### How do I run Windows containers on Docker Desktop on Windows Server 2016?
+### How do I run Windows containers on Docker Desktop on Windows Server?
 
-See [About Windows containers and Windows Server 2016](/install/windows/docker-ee/#about-docker-ee-containers-and-windows-server).
+See [About Windows containers and Windows Server](/install/windows/docker-ee/#about-docker-ee-containers-and-windows-server).
 
 A full tutorial is available in [docker/labs](https://github.com/docker/labs){: target="_blank" class="_"} at
 [Getting Started with Windows Containers](https://github.com/docker/labs/blob/master/windows/windows-containers/README.md){: target="_blank" class="_"}.
 
 ### Why is Windows 10 Home not supported?
 
-Docker Desktop for Windows requires the Hyper-V Windows feature which is not
-available on Home-edition.
+Docker Desktop requires the Hyper-V feature which is not available in the Windows Home edition.
 
 ### Why is Windows 10 required?
 
-Docker Desktop for Windows uses Windows Hyper-V. While older Windows versions have
-Hyper-V, their Hyper-V implementations lack features critical for Docker Desktop for
-Windows to work.
+Docker Desktop uses the Windows Hyper-V features. While older Windows versions have Hyper-V, their Hyper-V implementations lack features critical for Docker Desktop to work.
 
-### Why does Docker Desktop for Windows fail to start when firewalls or anti-virus software is installed?
+### Why does Docker Desktop fail to start when firewalls or anti-virus software is installed?
 
 Some firewalls and anti-virus software might be incompatible with Hyper-V and
 some Windows 10 builds (possibly, the Anniversary Update), which impacts Docker
-Desktop for Windows. See details and workarounds in
+Desktop. See details and workarounds in
 [Docker fails to start when firewall or anti-virus software is installed](troubleshoot#docker-fails-to-start-when-firewall-or-anti-virus-software-is-installed)
 in [Troubleshooting](troubleshoot).
 
 ## How do I uninstall Docker Toolbox?
 
-You might decide that you do not need Toolbox now that you have Docker Desktop for
-Windows, and want to uninstall it. For details on how to perform a clean
-uninstall of Toolbox on Windows, see
-[How to uninstall Toolbox](/toolbox/toolbox_install_windows#how-to-uninstall-toolbox) in the Toolbox Windows topics.
+Now that you have Docker Desktop, you might decide that you do not need Toolbox anymore. For details on how to perform a clean uninstall of Toolbox on Windows, see [How to uninstall Toolbox](/toolbox/toolbox_install_windows#how-to-uninstall-toolbox) in the Toolbox Windows topics.
```

Comment:
Should we add a statement on why they may no longer want to use Toolbox?

Reply:
Added legacy Desktop Toolbox for clarity